### PR TITLE
Print error message from request, if clipboardRequest return an error

### PR DESCRIPTION
**Pull request by cekk**

### CODE REVIEW

**Review by giuliaghisini — APPROVED** [review]

**Comment by stevepiercy** [comment]
@cekk change log, please!

**Comment by stevepiercy** [comment]
https://www.ueu.eus/ is suffering intermittent ISE 500 errors. Is that one of @erral's sites?

**Comment by erral** [comment]
it is. we are looking at it

**Review by pnicolli — APPROVED** [review]

**Comment by cekk** [comment]
> @cekk change log, please!

You're right..i didn't found the news folder with this new structure ;)

Added it

File: packages/volto/news/5932.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Print error message from request in toast, if clipboardRequest return an error. @cekk
```
Comment (stevepiercy) [inline packages/volto/news/5932.feature:1]:
```suggestion
Print error message from request in toast, if `clipboardRequest` return an error. @cekk
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
MyST syntax enhancement

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@cekk indeed @jensens also stumbled on it, and we updated the docs in hopes it made it easier to find in [Change log entry](https://6.docs.plone.org/contributing/index.html#change-log-entry). Sorry I didn't give that link, as I assumed you already knew.

Also @erral says a bad bot was hammering their site, and it should pass now.

If it's green, let's merge!

**Comment by stevepiercy** [comment]
Acceptance Tests / Core Blocks - Listing (18.x) (push) is flaky. I restarted it.

**Comment by stevepiercy** [comment]
All green!

**Pull request merged**